src/com/github/Kraken3/AFKPGC/EventHandlers.java

```diff
@@ -46,10 +46,6 @@ public void onPlayerMoveEvent(PlayerMoveEvent event) {
 		registerActivity(event.getPlayer());
 	}	
 	@EventHandler
-	public void onPlayerChatEvent(PlayerChatEvent event) {

```

Comment:
Wouldn't the non moving bots be kicked for not moving when the tick is low? I thought that was one of the conditions it looked for.